### PR TITLE
feat: Add invokeMode to FunctionUrl.

### PR DESCRIPTION
**Pull request by grasmash**
Lambda has released a new invokeMode parameter for Function URL, enabling response to be buffered or streamed back from the function. This change introduces the new parameter to the FunctionUrl class following the patterns established by existing parameters like `FunctionUrlCorsOptions`.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/aws-cdk-lib/aws-lambda/lib/function-url.ts [hunk 1]
```diff
@@ -59,6 +59,23 @@ export enum HttpMethod {
...
+
+  /**
+   *  Invocation results are available when the payload is complete.
+   */
+  BUFFERED = 'BUFFERED',
+
+  /**
+   * Your function streams payload results as they become available.
+   */
+  STREAMED = 'STREAMED',
```
Comment (valerena) [inline packages/aws-cdk-lib/aws-lambda/lib/function-url.ts:76]: This should probably be [`RESPONSE_STREAM`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-url.html#cfn-lambda-url-invokemode)

**Comment by grasmash** [comment]
I'm not sure this actually requires a new integration test .. I see no existing integration tests for Function URLs. Is that right?

**Comment by grasmash** [comment]
The existing function URL test has been updated. I don't think this requires a new test. Therefore, making an Exemption Request.

**Comment by grasmash** [comment]
The existing function URL test has been updated. I don't think this requires a new test. Therefore, making an Exemption Request.

**Comment by watany-dev** [comment]
私のPRと重複した内容のようです

https://github.com/aws/aws-cdk/pull/25375

**Comment by grasmash** [comment]
Yes and yours has better testing :) I'll close this.

**Pull request closed without merging**